cvat/apps/dataset_manager/bindings.py

```diff
@@ -605,7 +576,180 @@ def match_frame_fuzzy(self, path):
+                ("id", str(self._db_job.id)),
+                ("size", str(len(self))),
+                ("mode", self._db_task.mode),
+                ("overlap", str(self._db_task.overlap)),
+                ("bugtracker", self._db_task.bug_tracker),
+                ("created", str(timezone.localtime(self._db_task.created_date))),
+                ("updated", str(timezone.localtime(self._db_job.updated_date))),
+                ("subset", self._db_task.subset or dm.DEFAULT_SUBSET_NAME),
+                ("start_frame", str(self._db_data.start_frame + db_segment.start_frame * self._frame_step)),
+                ("stop_frame", str(self._db_data.start_frame + db_segment.stop_frame * self._frame_step)),
+                ("frame_filter", self._db_data.frame_filter),
+                ("segments", [
+                    ("segment", OrderedDict([
+                        ("id", str(db_segment.id)),
+                        ("start", str(db_segment.start_frame)),
+                        ("stop", str(db_segment.stop_frame)),
+                        ("url", "{}/api/jobs/{}".format(self._host, self._db_job.id))])),
+                ]),
+                ("owner", OrderedDict([
+                    ("username", self._db_task.owner.username),
+                    ("email", self._db_task.owner.email)
+                ]) if self._db_task.owner else ""),
+
+                ("assignee", OrderedDict([
+                    ("username", self._db_job.assignee.username),
+                    ("email", self._db_job.assignee.email)
+                ]) if self._db_job.assignee else ""),
+            ])),
+            ("dumped", str(timezone.localtime(timezone.now()))),
+        ])
+
+        if self._label_mapping is not None:
+            self._meta[JobData.META_FIELD]["labels"] = CommonData._convert_db_labels(self._label_mapping.values())
+
+        if hasattr(self._db_data, "video"):
+            self._meta["original_size"] = OrderedDict([
+                ("width", str(self._db_data.video.width)),
+                ("height", str(self._db_data.video.height))
+            ])
+
+    def __len__(self):
+        segment = self._db_job.segment
+        return segment.stop_frame - segment.start_frame + 1
+
+    def _get_queryset(self):
+        return self._db_data.images.filter(frame__in=self.abs_range)
+
+    @property
+    def abs_range(self):
+        segment = self._db_job.segment
+        step = self._frame_step
+        start_frame = self._db_data.start_frame + segment.start_frame * step
+        stop_frame = self._db_data.start_frame + segment.stop_frame * step + 1
+
+        return range(start_frame, stop_frame, step)
+
+    @property
+    def rel_range(self):
+        segment = self._db_job.segment
+        return range(segment.start_frame, segment.stop_frame + 1)
+
+    @property
+    def start(self):
+        segment = self._db_job.segment
+        return segment.start_frame
+
+    @property
+    def offset(self):
+        return self.start
```

Comment:
Why do both `offset` and `start` need to exist? Seems like they are the same thing.